static/app/components/globalDrawer/useDrawerResizing.tsx

```diff
@@ -28,6 +29,7 @@ interface UseDrawerResizingResult {
 export function useDrawerResizing({
   drawerKey,
   drawerWidth,
+  useFixedWidth = false,
```

Comment:
tbh it might be more worthwhile to allow disabling this hook entirely. Passing in a fixed value for a resizing function is a bit confusing.

 I also think it'd be worthwhile since the current implementation  hinges on the presence of a `drawerKey` which is a string, but we can make it explicit that we want to disable resizing by passing in `enabled: false` or something, similar to how our `useApiQuery`s work.

Reply:
It'd let us remove `useFixedWidth` altogether 👍

Reply:
I also think it makes sense for not all drawers to be resizable in general so a drawer option like `resizable?: boolean` would be helpeful